ci/cd/utils/test_artifact_repository.py

```diff
@@ -0,0 +1,494 @@
+import unittest
```

Comment:
Would you like to add that to one of our CI pipelines?

Reply:
Do you think it would make sense? Feels kinda unrelated to what's there somehow

Reply:
Create a new category then :)

Reply:
Added